docs/ops/pooling/AdaptiveAvgPool_8.md

```diff
@@ -0,0 +1,80 @@
+## AdaptiveAvgPool<a name="AdaptiveAvgPool"></a> {#openvino_docs_ops_pooling_AdaptiveAvgPool_8}
+
+**Versioned name**: *AdaptiveAvgPool-8*
+
+**Category**: *Pooling*
+
+**Short description**: Applies average pooling with adaptive kernel size over the input.
+
+**Detailed description**: This operation calculates the output based on the 1st input and `output_size` determined by the 2nd input.
+The kernel dimensions are calculated using the following formulae for the `NCDHW` input case:
+
+\f[
+\begin{array}{lcl}
+d_{start} &=& floor(i*D_{in}/D_{out})\\
+d_{end}   &=& ceil((i+1)*D_{in}/D_{out})\\
+h_{start} &=& floor(j*H_{in}/H_{out})\\
+h_{end}   &=& ceil((j+1)*H_{in}/H_{out})\\
+w_{start} &=& floor(k*W_{in}/W_{out})\\
+w_{end}   &=& ceil((k+1)*W_{in}/W_{out})
+\end{array}
+\f]
+
+The output is calculated following this formula:
+
+\f[
+Output(i,j,k) = \frac{Input[d_{start}:d_{end}, h_{start}:h_{end}, w_{start}:w_{end}]}{(d_{end}-d_{start})*(h_{end}-h_{start})*(w_{end}-w_{start})}
+\f]
```

Comment:
IMHO, now I am afraid that these formulas look not so obvious and can be useless for developers.
I propose:
1) I think that these formulas can be simplified by just describing pooling process for 3D tensor. For higher rank tensor it is generalized simply.
2) Avoid latex denotations that routine all stuff here because this case is simple. Also you use `/` and `\frac` for the same operations. I propose to use just `/`.